File: web3rpc/rpc-specs/code-samples/python/kaia/configuration/isConsoleLogEnabled.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.is_console_log_enabled()

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/configuration/isConsoleLogEnabled.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  provider.klay
    .isConsoleLogEnabled({}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
